Cargo.toml

```diff
@@ -34,8 +34,8 @@ futures-executor = "0.3.21"
 futures-util = { version = "0.3.26", default-features = false, features = ["alloc"] }
 http = "0.2.6"
 itertools = "0.11.0"
-ruma = { git = "https://github.com/ruma/ruma", rev = "8b4ece2b0fa31be21eb1273a6e6bf32e09909480", features = ["client-api-c", "compat-upload-signatures", "compat-user-id"] }
-ruma-common = { git = "https://github.com/ruma/ruma", rev = "8b4ece2b0fa31be21eb1273a6e6bf32e09909480" }
+ruma = { git = "https://github.com/ruma/ruma", rev = "4295563c73a529188793dc320e62994e38c4941c", features = ["client-api-c", "compat-upload-signatures", "compat-user-id"] }
```

Comment:
Reminder: Needs to be bumped after your Ruma PR is merged.

Reply:
Bumped now.